collection-scripts/gather_version

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# load shared functions and data when not sourced
+if [[ -z "$DIR_NAME" ]]; then
+    CALLED=1
+    DIR_NAME=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+    source "${DIR_NAME}/common.sh"
+fi
+
+function ocp_version() {
+    oc_version=$(oc version -o json | jq -r .openshiftVersion)
+    if [[ -n $oc_version ]] ; then
+        echo "${oc_version}"
+    else
+        echo "0.0.0-unknown"
+    fi
+}
+
+function log_version() {
+    echo "openstack-k8s-operators/openstack-must-gather"> "${BASE_COLLECTION_PATH}/version"
+    ocp_version >> "${BASE_COLLECTION_PATH}/version"
```

Comment:
We might want to consolidate `log_version` and `ocp_version` in a single function, but that would just be a personal preference that doesn't block this patch from a functional point of view.

Reply:
I have changed that